src/repos/UserRepo.ts

```diff
@@ -64,30 +64,14 @@ const updateUser = async (user: User, userFields: UserUpdateFields): Promise<boo
 const getUserByNetID = async (netID: string): Promise<User | undefined> => {
   const user = await db().findOne({
     where: { netID },
-    relations: [
-      'matches',
-      'matches.users',
-      'matches.schedule',
-      'matches.schedule.times',
-      'clubs',
-      'interests',
-      'major',
-    ],
+    relations: ['availabilities', 'clubs', 'interests', 'major'],
```

Comment:
@chalo2000 let's add `availabilities.users` here and in `getUsers`. That way, we can get rid of the `eager: true` option in the `Availability` entity (just for consistency's sake). I believe this should also work.

Reply:
actually POG